src/core/mac/sub_mac.hpp

```diff
@@ -553,14 +554,20 @@ class SubMac : public InstanceLocator, private NonCopyable
 
 #if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
     enum : uint32_t{
-        kCslSampleWindow = OPENTHREAD_CONFIG_CSL_SAMPLE_WINDOW *
-                           kUsPerTenSymbols, ///< The SSED sample window in units of microseconds.
+        kMinCslWindow = OPENTHREAD_CONFIG_CSL_MIN_RECEIVE_ON, ///< CSL receive window for the longest possible
+                                                              ///< frame and ack duration.
         kCslReceiveTimeAhead =
             OPENTHREAD_CONFIG_CSL_RECEIVE_TIME_AHEAD, ///< CSL receivers would wake up `kCslReceiveTimeAhead` earlier
                                                       ///< than expected sample window. The time is in unit of 10
                                                       ///< symbols.
     };
 
+    enum : uint8_t{
+        kCslCrystalPpm = OPENTHREAD_CONFIG_CSL_CRYSTAL_PPM, ///< Accuracy of the crystal used to schedule CSL
```

Comment:
Makes sense, however _The device's XTAL accuracy, in ppm_ is not clear for me in this situation. It might happen that for CSL operations a different precision is achieved. In any case I will use it now and add improvements when this changes in the future.

Reply:
I replaced this value with `otPlatRadioGetCslAccuracy` that would allow for platforms to implement real time corrections on the accuracy.